src/pages/trade/ui/route-book.tsx

```diff
@@ -1,60 +1,212 @@
+import React, { useState } from 'react';
 import { useBook } from '../api/book';
 import { observer } from 'mobx-react-lite';
-import { RouteBookResponse } from '@/shared/api/server/book/types';
+import { RouteBookResponse, Trace } from '@/shared/api/server/book/types';
+import { ChevronRight } from 'lucide-react';
+import { getSymbolFromValueView } from '@penumbra-zone/getters/value-view';
+import { usePathSymbols } from '@/pages/trade/model/use-path.ts';
 
-const RouteBookLoadingState = () => {
+import { Tabs } from '@penumbra-zone/ui/Tabs';
+
+const HopCount = ({ count }: { count: number }) => {
+  return (
+    <span className={count === 0 ? 'text-white' : 'text-[#F49C43]'}>
+      {count === 0 ? 'Direct' : `${count} ${count === 1 ? 'Hop' : 'Hops'}`}
```

Comment:
issue: For direct, it should have two hops right? The first is the starting token and the second is the ending token. In this case, I think it likely should be something like:
1. Direct, if hop === 2
2. Else, hop = traces.hops.length - 1